### PR TITLE
[MNG-5760] --resume feature (part 2)

### PR DESCRIPTION
**Pull request by mthmulders**
Added [the testcases for `--resume`](https://github.com/apache/maven-integration-testing/pull/61) to the suite. Also added a few edge cases for that feature to the existing tests.

### CODE REVIEW

File: core-it-suite/src/test/java/org/apache/maven/it/MavenITmng5760ResumeFeatureTest.java [hunk 1]
```diff
@@ -40,19 +40,26 @@
...
 
     public MavenITmng5760ResumeFeatureTest() throws IOException {
         super( "[3.7.0,)" );
-        this.testDir = ResourceExtractor.simpleExtractResources( getClass(), "/mng-5760-resume-feature" );
+        this.parentDependentTestDir = ResourceExtractor.simpleExtractResources( getClass(),
+                "/mng-5760-resume-feature/parent-dependent" );
+        this.parentIndependentTestDir = ResourceExtractor.simpleExtractResources( getClass(),
+                "/mng-5760-resume-feature/parent-independent" );
+        this.noProjectTestDir = ResourceExtractor.simpleExtractResources( getClass(),
+                "/mng-5760-resume-feature/no-project" );
```
Comment (MartinKanters) [inline core-it-suite/src/test/java/org/apache/maven/it/MavenITmng5760ResumeFeatureTest.java:54]: I think we are missing these files.
After squashing and merging this branch to Apache, the GitHub Actions pipeline was failing:
https://github.com/apache/maven/runs/872356902?check_suite_focus=true
`testShouldSkipSuccessfulProjects (java.lang.IllegalArgumentException: Resource not found: /mng-5760-resume-feature/no-project`

**Comment by MartinKanters** [comment]
Merged in [2f1a9ef3a205dfe89c4a53c419c8f3f6c27c2f96](https://github.com/apache/maven-integration-testing/commit/2f1a9ef3a205dfe89c4a53c419c8f3f6c27c2f96)
